src/code/z_actor.c

```diff
@@ -106,11 +135,30 @@ void Actor_TitleCardContextInit(GlobalContext* ctxt, TitleCardContext* titleCtxt
     titleCtxt->alpha = 0;
 }
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_TitleCardCreate.asm")
+void Actor_TitleCardCreate(GlobalContext* ctxt, TitleCardContext* titleCtxt, u32 texture, s16 param_4, s16 param_5, u8 param_6, u8 param_7) {
+    titleCtxt->texture = texture;
+    titleCtxt->unk4 = param_4;
+    titleCtxt->unk6 = param_5;
+    titleCtxt->unk8 = param_6;
+    titleCtxt->unk9 = param_7;
+    titleCtxt->fadeOutDelay = 80;
+    titleCtxt->fadeInDelay = 0;
+}
 
 GLOBAL_ASM("asm/non_matchings/z_actor//Actor_Nop800B5E50.asm")
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_TitleCardUpdate.asm")
+void Actor_TitleCardUpdate(GlobalContext* ctxt, TitleCardContext* titleCtxt) {
+    if (DECR(titleCtxt->fadeInDelay) == 0) {
+        if (DECR(titleCtxt->fadeOutDelay) == 0) {
+            Lib_StepTowardsCheck_s(&titleCtxt->alpha, 0, 30);
+            Lib_StepTowardsCheck_s(&titleCtxt->color, 0, 70);
+        }
+        else {
```

Comment:
Put the else on the same line as the closing brace.
```suggestion
        } else {
```